Ball.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+public class Ball {
+    Colors color;
+    int number;
+    @Override
+    public String toString() {
+        return "Ball " + color + " " + number;

```

Comment:
here toString will be called for enum by default. Revisit checklist to see why it's undesirable